ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/linear_optimisation/fillers/MaxLoopFlowFillerTest.java

```diff
@@ -34,20 +38,27 @@ public void setUp() {
     }
 
     @Test
-    public void testFill() {
+    public void testFillCaseInitialLoopFlowIsLowerThanInputThreshold() {
 
-        CnecLoopFlowExtension cnecLoopFlowExtension = new CnecLoopFlowExtension(0.0, Unit.PERCENT_IMAX);
+        CnecLoopFlowExtension cnecLoopFlowExtension = new CnecLoopFlowExtension(100.0, Unit.MEGAWATT);
         cnecLoopFlowExtension.setLoopFlowConstraintInMW(100.0);
         cnecLoopFlowExtension.setLoopflowShift(49.0);
         cnec1.addExtension(CnecLoopFlowExtension.class, cnecLoopFlowExtension);
+        String testVariant = "test-variant";
+        ResultVariantManager resultVariantManager = new ResultVariantManager();
+        crac.addExtension(ResultVariantManager.class, resultVariantManager);
+        crac.getExtension(ResultVariantManager.class).createVariant(testVariant);
+        crac.getExtension(ResultVariantManager.class).setInitialVariantId(testVariant);
+        cnec1.getExtension(CnecResultExtension.class).getVariant(testVariant).setFlowInMW(85.);
```

Comment:
shouldn't be required by the filler - its look like old code which was relevant in the version you show me yesterday but not anymore ;)